.github/workflows/publish.yml

```diff
@@ -37,3 +37,97 @@ jobs:
         run: npm publish --tag ${{ steps.npm_tag.outputs.npm_tag }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - name: Wait for 300 seconds
+        run: sleep 300s
+        shell: bash
+  deploy_chime_prod_demo:
+    needs: publish
+    name: Prod - Chime Client - Deploy the Serverless Meeting Demos
+    runs-on: ubuntu-latest
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_DEFAULT_OUTPUT: text
+      NAME: ChimeProd
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.PROD_AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}
+          aws-session-token: ${{ secrets.PROD_AWS_SESSION_TOKEN }}
+          aws-region: us-east-1
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Get current published version
+        id: npm_version
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "Current released version:" $current_version
+          echo ::set-output name=npm_version::$current_version
+      - name: Install SDK and verify npm version
+        run: |
+          cd demos/browser
+          npm uninstall amazon-chime-sdk-js
+          npm install amazon-chime-sdk-js@${{ steps.npm_version.outputs.npm_version }}
+      - name: Verify npm version
+        run: |
+          demo_current_version=$(.github/script/get-demo-browser-current-version)
+          echo "Current demo version:" $demo_current_version
+          if [[ *$demo_current_version* != "^#${{ steps.npm_version.outputs.npm_version }}" ]];
+          then
+            echo "Demo npm version is not the same as the current released version"
+          else
+            npm run build
+            npm run build --app=meetingReadinessChecker
+            script/github-action-awscli-installation
+            script/postrelease
+          fi
```

Comment:
Im wondering if it might be simpler just to add this to the matrix below. Itd make this file simpler. Anyways, we're still going to deploy all of the demos at the same time, I dont think it really matters that it needs its own job

Reply:
Yes, initially did that way, then thought its better to keep separate so whoever goes through it have an idea about which Meetings client is getting deployed. Also, before merging the CR we need to - 
1. Configure PROD_AWS_ACCESS_KEY and PROD_AWS_SECRET_ACCESS_KEY in Github secrets. 
2. Increase S3 bucket limit in Prod canary account.

Reply:
Updated, just creating a single job